lighthouserc.json

```diff
@@ -0,0 +1,34 @@
+{
+  "ci": {
+    "upload": {
+      "target": "temporary-public-storage"
+    },
+    "assert": {
+      "assertions": {
+        "categories:performance": ["warn", { "minScore": 0.6 }],
+        "categories:accessibility": ["error", { "minScore": 0.8 }],
+        "categories:best-practices": ["error", { "minScore": 0.8 }],
+        "categories:seo": ["error", { "minScore": 0.8 }],
+        "categories:pwa": ["warn", { "minScore": 0.0 }]
+      }
+    },
+    "collect": {
+      "numberOfRuns": 1,
+      "url": [
+        "http://localhost:3000/",
+        "http://localhost:3000/verify",
+        "http://localhost:3000/faq",
+        "http://localhost:3000/eta",
+        "http://localhost:3000/news",
+        "http://localhost:3000/learn",
+        "http://localhost:3000/event",
+        "http://localhost:3000/guidelines"
+      ],
```

Comment:
use glob pattern

Reply:
what do u mean? got example?

Reply:
wdyt to use .[js](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#file-structure) as config instead and import the `routes` into `url `?